coding_style_and_standards.md

```diff
@@ -88,7 +88,6 @@ documentation server which regenerates the docs on each request.
 ## Commits
 
 * Write a good commit message.  The format for a commit message is as follows.
-  Also [read more](http://goo.gl/w11us) on writing good commit messages.
```

Comment:
This link is dead and I have no idea what is was to see if there's a replacement. I tried searching for a good link to replace it, but I can't really find a good one.

Reply:
I think have the gist of the link details in the lines that follow anyway.